components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/dto/APIDTO.java

```diff
@@ -139,7 +139,7 @@ public static VisibilityEnum fromValue(String v) {
 return null;
         }
     }
-    @Scope(name = "apim:api_publish", description="", value ="")
+    @Scope(name = "{apim:api_publishAPI&#x3D;null}", description="", value ="")
```

Comment:
Regenerating code (do an `mvn clean install` from `components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common`) will restore this change